addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -584,6 +602,13 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
                 if (value) {
                     return value;
                 } else {
+                    if (params.fieldName === 'email_to') {
+                        // For email_to, we try to find a value in this order:
+                        // 1. The current value of the input
+                        // 2. The data-for value if it exists
+                        // 3. The default value (`defaultEmailToValue`)
+                        return this.dataForEmailTo || this.defaultEmailToValue;
+                    }
```

Comment:
@qsm-odoo I've adapted the code to your idea
> "if email_to has a value, show it, otherwise fallback on the data-for if any, otherwise show info@yourcompany.example.com

I just had to change the way the default value for email_to is used for it to work. Let me know what you think.